### PR TITLE
CircleCI: Add config.yml.

### PR DESCRIPTION
**Pull request by byohay**

### CODE REVIEW

File: lib/xcpretty/parser.rb [hunk 1]
```diff
@@ -568,7 +568,7 @@ def update_linker_failure_state(text)
       when LINKER_UNDEFINED_SYMBOL_LOCATION_MATCHER
         current_linker_failure[:body] << $1.rstrip
       when LINKER_DUPLICATE_SYMBOLS_LOCATION_MATCHER
-        current_linker_failure[:body] << "    " + $1.rstrip.split("/").last
```
Comment (barakwei) [inline lib/xcpretty/parser.rb]: keep it, fix the tests

Comment (barakwei) [reply lib/xcpretty/parser.rb]: maybe keep the rstrip if you want

Comment (byohay) [reply lib/xcpretty/parser.rb]: I thought that we want to show the full path of the file, not just the filename. The `rstrip`'s still there.

**Comment by byohay** [comment]
@barakwei answered.

File: lib/xcpretty/parser.rb (hunk 1 above)
Comment (barakwei) [reply lib/xcpretty/parser.rb]: Let's do just the file.

Comment (byohay) [reply lib/xcpretty/parser.rb]: Reverted.

**Comment by byohay** [comment]
@barakwei F&Sed.

**Comment by barakwei** [comment]
tests fail

**Pull request merged**